Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertTopography.cs

```diff
@@ -29,10 +30,15 @@ public ApplicationObject TopographyToNative(Topography speckleSurface)
         pts.Capacity += displayMesh.vertices.Count / 3;
         for (int i = 0; i < displayMesh.vertices.Count; i += 3)
         {
-          var point = new Geometry.Point(displayMesh.vertices[i], displayMesh.vertices[i + 1], displayMesh.vertices[i + 2], displayMesh.units);
-          pts.Add(PointToNative(point));
+          // add a check for duplicate points, if 'keepXYDuplicates' is false 
+          var ptTuple = (displayMesh.vertices[i], displayMesh.vertices[i + 1]);
+          if ( keepXYDuplicates || !pointTuplesList.Contains(ptTuple) )
+          {
+            pointTuplesList.Add(ptTuple);
+            var point = new Geometry.Point(displayMesh.vertices[i], displayMesh.vertices[i + 1], displayMesh.vertices[i + 2], displayMesh.units);
+            pts.Add(PointToNative(point));
+          }
```

Comment:
This works right now only because we're using [`Topography.Create`](https://www.revitapidocs.com/2022/824b6a30-f775-149f-2e6a-5198c5299d14.htm) with the overload that doesn't accept triangulation (see line 47 of this file).

There is an overload [that does accept triangulation](https://www.revitapidocs.com/2022/5fa47ddd-7dbd-ae8a-69bc-9e260ec56d99.htm), and has been brought up in the forum in the past, but removing duplicates will screw around with triangulation too.

Maybe out of scope for this PR but felt it necessary to mention this.